TACC/Core-Portal
# task/FP-1045, bug/FP-1048, task/FP-1049: Various jobs fixes

## Overview: ##

- Fixes an issue where interactive sessions were ready but not visible because of lagging Tapis webhooks
- Allows any tar, gz, or zip extension to be extracted with the extract utility button
- Applies tapis URI validation as default validation for Tapis input fields

## Related Jira tickets: ##

* [FP-1045](https://jira.tacc.utexas.edu/browse/FP-1045)
* [FP-1048](https://jira.tacc.utexas.edu/browse/FP-1048)
* [FP-1049](https://jira.tacc.utexas.edu/browse/FP-1049)

## Testing Steps: ##
1. Launch an interactive job and ensure that you can click "Open Session" whenever the Session is Ready webhook comes in, no matter the status of the job (as long as it's not terminal or finishing).
2. Confirm you can use the "Extract" button utility in the datafiles area on any .zip, .tar, .gz, .tar.gz, or .tgz extension.
3. Confirm you get a validation error if you type anything in a file input field that does not start with `agave://`
